types/d3-scale/d3-scale-tests.ts

```diff
@@ -133,7 +135,9 @@ clampFlag = linearScaleNumber.clamp();
 
 // unknown(...) -----------------------------------------------------------------
 
-linearScaleNumber = linearScaleNumber.unknown(5);
+linearScaleNumberUnknownNumber = linearScaleNumber.unknown(5);
+linearScaleNumberUnknownString = linearScaleNumber.unknown('test');
+const numOrString: number | string = linearScaleNumberUnknownString(3);
 const linearScaleNumberUnknownValue = linearScaleNumber.unknown();
```

Comment:
do you want to assert a type on this (`: number`)?

Reply:
Sure.

Reply:
Except it will be `undefined`.

Reply:
Right! 👍